http/src/v0/swarm.rs

```diff
@@ -162,14 +162,18 @@ pub fn addrs_local<T: IpfsTypes>(
 
 #[derive(Debug, Deserialize)]
 struct DisconnectQuery {
-    arg: Multiaddr,
+    arg: String,
```

Comment:
I wonder if the `crate::v0::support::StringSerialized<MultiaddrWithPeerId>` could be used here? It'll probably change the error message to something ackward.

Reply:
Yeah, it works :+1:.

Reply:
...but the error message is less verbose, so let's leave it as `String` for now, as discussed.